release.sh

```diff
@@ -204,14 +202,27 @@ else
     git commit -n -m "v$VERSION"
     git diff --stat --exit-code # check for uncommitted changes
     git tag -a "v$VERSION" -m "v$VERSION"
-    git push origin "v$VERSION"
+    git push "$ORIGIN" "v$VERSION"
 
     # Update npm (cdnjs update automatically)
-    yarn publish --new-version "${VERSION}"
+    # Fallback to npm publish, if yarn cannot authenticate, e.g., 2FA
+    yarn publish --new-version "$VERSION" || npm publish
```

Comment:
Will this work in DRYRUN mode? I guess the first operation will always succeed, but I'd feel a little safer if we defined a function `npm() { echo "npm $*" }`.

Reply:
@edemaine I've added `npm() { echo "npm $*" }`.